### PR TITLE
[FIX] dash filters data

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
[FIX](https://bcdevex.atlassian.net/browse/FIX)

Objective:

- fix the request to get dashboard filters

### CODE REVIEW

File: frontend/src/pages/dashboard/constants.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const dashboardToggle = {
   name: DashboardFilterNames.SHOW_INACTIVE,
   label: 'Show Inactive',
 };
-export interface WorkLocation {
+export interface LocationInterface {
```
Comment (fw-noel) [inline frontend/src/pages/dashboard/constants.ts:45]: Why this change? We don't tend to name our interfaces `xInterface` (I see now that there's a few others that I missed), so curious as to why here!

**Review by fw-noel — DISMISSED** [review]

File: frontend/src/pages/dashboard/constants.ts (hunk 1 above)
Comment (chelsea-EYDS) [reply frontend/src/pages/dashboard/constants.ts:45]: WorkLocation was already an enum, and then I wanted to make it more generic since we're using this for both home and work, and then I just didn't remove the Interface. I add interface when there is a naming conflict. I supposed I could just alias the import but it makes searching easier if they're named differently. I will fix :)

**Review by fw-noel — APPROVED** [review]
